crate_universe/3rdparty/crates/BUILD.linux-raw-sys-0.3.8.bazel

```diff
@@ -48,6 +48,9 @@ rust_library(
         "@rules_rust//rust/platform:i686-unknown-linux-gnu": [
             "errno",  # i686-unknown-linux-gnu
         ],
+        "@rules_rust//rust/platform:s390x-unknown-linux-gnu": [
```

Comment:
I noticed on https://github.com/bazelbuild/rules_rust/pull/2841 that this change goes away when I regenerate dependencies. How was this produced?

Reply:
@UebelAndre - any specific reason to remove this?

Reply:
These files are generated and if there is a change made to them they should be made in a way that regenerates into the file. The removal feels more like a miss where something was not hooked up correctly and I was wondering how these came to be in the first place.